### ISSUE TITLE
Duplicating calendars when reading calendar is impossible

### ISSUE THREAD

**Issue by xatr0z** [gggard/AndroidCaldavSyncAdapater#152]
I struggled to get caldavsync 1.8.0 and 1.8.1 working with Owncloud 5 on my Galaxy S GT-I9000 with Cyanogenmod 10.2.0. 

Both my own setup and demo.owncloud.org would keep adding the calendar every minute or so (I guess every time the app tries to sync). In a few minutes I end up with lots of duplicate calendars (in calendars to display and -sync). Removing them from the synced calendars is difficult, because they reappear again and again. Besides that I also got multiple "caldav sync error (general)" errors.

I found the culprit to be Privacy Guard, a feature in the latest Cyanogenmod builds to finetune control of permissions for apps (I enabled its function to use privacy guard by default for new apps, which is turned off after installation). In the settings for CalDAV Sync Adapter I saw that 'Modify Calendar' was allowed, but 'Reading Calendar' (and a lot of other operations) was blocked.

Tl;dr by allowing 'Reading Calendar' or turning off Privacy Guard for this app errors and forkbombing calendars will disappear. It's working perfectly fine now, with two-way sync.

Obviously you can't do anything about privacy guard. But maybe the duplicating calendars issue is fixable with an extra check somewhere?
